scripts/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# Change Log
+
+All notable changes to this module will be documented centrally in the [SAP Cloud SDK release notes](https://sap.github.io/cloud-sdk/docs/js/release-notes-sap-cloud-sdk-for-javascript-and-typescript/).
```

Comment:
This is not a script - I would rather put it into test-resources.

Reply:
moved